doc/source/getting_started/comparison/comparison_with_spreadsheets.rst

```diff
@@ -365,6 +365,27 @@ In Excel, there are `merging of tables can be done through a VLOOKUP
 * It will include all columns from the lookup table, instead of just a single specified column
 * It supports :ref:`more complex join operations <merging.join>`
 
+GroupBy
+-------
+
+In Excel, this can be done by using the `Query Editor <https://support.microsoft.com/en-us/office/introduction-to-the-query-editor-power-query-1d6cdb63-bf70-4ae8-a7d5-6ae9547004d9>`_.
```

Comment:
This page says it will be retired soon, perhaps https://support.microsoft.com/en-us/office/about-power-query-in-excel-7104fbee-9e62-4cb9-a02e-5bfb1a6c536a is better.

Reply:
Done.